README.md

```diff
@@ -8,7 +8,8 @@ This code is open source software licensed under the [Apache 2.0 License]("http:
 
 ## Endpoints
 
-```POST /ctc/registerncf/v1```
+```POST /ncts5/noticeofcrossingfrontiergb/v1```
+```POST /ncts5/noticeofcrossingfrontierxi/v1```
```

Comment:
Would you duplicate the request please (L14 to L34), one for gb one for xi as the request body is not same.